.flowconfig

```diff
@@ -18,6 +18,7 @@
 .*/node_modules/@react-native-community/cli/.*/.*
 
 [include]
+../../node_modules
```

Comment:
Wouldn't it collide with line 9?
https://github.com/callstack/react-native-slider/blob/470288ca40b77cd538ed1c8101fe63f332d47d02/.flowconfig#L9